swaggerpy/async_http_client.py

```diff
@@ -21,12 +22,9 @@
 from twisted.web.client import Agent
 from twisted.web.client import FileBodyProducer
 from twisted.web.http_headers import Headers
-from yelp_uri import urllib_utf8
```

Comment:
fixed but had to port some helpers inside. still better than preventing people from using this because of yelp_uri.